main.tf

```diff
@@ -1,6 +1,31 @@
+module "sqs_queue_dlq" {
+  source                            = "terraform-aws-modules/sqs/aws"
+  version                           = "~> 3.2.0"
+  create                            = length(var.dlq_options) > 0 ? true : false
+  name                              = lookup(var.dlq_options, "name", "${var.name}_dlq")
+  fifo_queue                        = var.fifo_queue
+  visibility_timeout_seconds        = lookup(var.dlq_options, "visibility_timeout_seconds", null)
+  message_retention_seconds         = lookup(var.dlq_options, "message_retention_seconds", null)
```

Comment:
These should have defaults... see ticket

Reply:
If we set it to null it will take defaults from actual module https://github.com/terraform-aws-modules/terraform-aws-sqs/blob/master/variables.tf#L19

Reply:
Added as 14 days